src/templates/home-page/BTCRole/index.tsx

```diff
@@ -1,22 +1,33 @@
 import { FC } from "react"
+import { Divider, SimpleGrid } from "@chakra-ui/react"
 import SectionTemplate, { RoleTemplateProps } from "../SectionTemplate"
-import { TBTCPartnerImage, TBTCPartners } from "./tBTCPartners"
+import { renderPartners, TBTCPartner } from "../../../components/tBTCPartners"
+import { H4 } from "../../../components"
 
 interface BTCRoleProps {
-  minters: TBTCPartnerImage[]
-  guardians: TBTCPartnerImage[]
+  partners: TBTCPartner[]
 }
 
 export const BTCRole: FC<RoleTemplateProps & BTCRoleProps> = ({
-  minters,
-  guardians,
+  partners,
   ...props
-}) => {
-  return (
-    <>
-      <SectionTemplate {...props}>
-        <TBTCPartners minters={minters} guardians={guardians} mt={"6rem"} />
-      </SectionTemplate>
-    </>
-  )
-}
+}) => (
+  <>
+    <SectionTemplate {...props}>
+      <Divider my="5.5rem" />
+      <H4 color="gray.300">tBTC is backed by the following partners.</H4>
+      <SimpleGrid
+        mt="4.5rem"
+        spacingY="5.5rem"
+        justifyContent="space-between"
```

Comment:
Non-blocking:

I think it looks ok now except the smallest screen when there are two icons in a row:

<img width="676" alt="image" src="https://github.com/threshold-network/splash-page/assets/40306834/c127bd66-9227-4f02-b833-4436a244d187">

There is too much space between the rows imo and it looks a little bit off to me. I think we could keep `space-around` in that case so I would suggest to change it to:

```suggestion
       justifyContent={{ base: "space-around", md: "space-between" }}
```

Result:
<img width="631" alt="image" src="https://github.com/threshold-network/splash-page/assets/40306834/7458d7f0-19a8-4c54-a822-75461c2c2078">

Reply:
ebf4298